dev/tests/unit/Magento/FunctionalTestFramework/DataGenerator/Handlers/PersistedObjectHandlerTest.php

```diff
@@ -32,6 +32,31 @@ public function setUp()
         TestLoggingUtil::getInstance()->setMockLoggingUtil();
     }
 
+    public function testCreateInvalidEntity()
```

Comment:
There are other invalid entity test cases at the bottom of the file. How about rename this test with something like "testCreateEntityWithNonExistingName"?